docs/FAQ.md

```diff
@@ -91,8 +91,8 @@ carry out a TrenchBoot "Secure Launch". This customized GRUB introduces the
 can be found at
 [https://github.com/3mdeb/meta-trenchboot](
     https://github.com/3mdeb/meta-trenchboot).
-The custom GRUB is however unfinished and while it is running on AMD, it is not
-yet functional with Intel processors.
+The custom GRUB works on Linux and Multiboot2 protocols for both Intel and AMD
+devices, but only in case of legacy boot, without support for UEFI.
```

Comment:
Notice this paragraph says that GRUB works on Linux protocols, not on Linux. I haven't heard about any Linux protocols though. Maybe it should say that it "works with Linux and on Multiboot2 protocol"?

Reply:
What do you think? https://github.com/TrenchBoot/TrenchBoot.github.io/pull/41/commits/0151b8ba63954daca19d4b76590dae0684f6a449

Reply:
Here "Linux protocol" means Linux boot protocol.  Maybe "boot" should be added to reduce confusion.

```suggestion
The custom GRUB works with Linux and Multiboot2 boot protocols for both Intel and AMD
devices, but only in case of legacy boot, without support for UEFI (although it's available on
a separate branch).
```

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/0151b8ba63954daca19d4b76590dae0684f6a449..382f343dd9947180a69b693d3eb0720b69cfa754#diff-0bb42a85d04c8b7c9b659307979fd43e8f6e084283a300bfff959adfcbddfd17R94-R96